packages/components/src/components/expansion-panel/array-renderer.jsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+const ArrayRenderer = (array, open_ids, setOpenIds) => {
+    const onArrayItemClick = id => {
+        if (open_ids.includes(id)) {
+            setOpenIds(open_ids.filter(open_id => open_id !== id));
+        } else {
+            setOpenIds([...open_ids, id]);
+        }
+    };
+
+    return (
+        <div>
+            {array.map((item, index) => {
+                if (item && item.value !== undefined && item.value !== null && Array.isArray(item.value)) {
+                    return (
+                        <div key={index} className='dc-expansion-panel__content-array'>
+                            <div
+                                className={classNames('dc-expansion-panel__content-array', {
+                                    'dc-expansion-panel__content-active': open_ids.includes(item.id),
+                                })}
+                            >
+                                <span className='dc-expansion-panel__content-array-item-index'>{index + 1}: </span>(
```

Comment:
Nit, to make it clearer there's a string being rendered here you could use string literals, e.g.

```JS
<span className='dc-expansion-panel__content-array-item-index'>
    {`${index + 1}:`}
</span>

{`(${item.value.length})`}
```

Reply:
This is a nice idea. I implemented it.